docs/tutorial/02-listening.rst

```diff
@@ -0,0 +1,43 @@
+Listening to a graph
+====================
+
+Building a story graph is one thing, but just as important is listening to the graph.
+Gencaster provides a player for a story graph via a website, which makes it possible to distribute the content of the story graph via mobile devices to any place where there is mobile coverage.
```

Comment:
Gencaster provides a player for a story graph via a website, which makes it possible to distribute the content of the story graph to any internet-enabled device.